UrbanAirship.NETStandard.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
    <metadata>
       <id>urbanairship.netstandard</id>
-      <version>11.1.0</version>
+      <version>12.0.0</version>
```

Comment:
what are the rules we're following for updating the versions for netstandard and portable again? I'm confused why this is a major version bump.

Reply:
For .NETStandard and PCL, we bump it at least as much as we bumped the platform-specific. So, if iOS got a major and Android got a minor, .NETStandard and PCL get a major.

Reply:
But it was at 11.1.0 for 13.0.4?

Reply:
was just just an oversight last time around?

Reply:
Sort of an oversight. I didn't go through all of the release steps last time around because we didn't release.